src/bm.h

```diff
@@ -653,7 +658,7 @@ void basm_push_deferred_operand(Basm *basm, Inst_Addr addr, String_View label)
 Word number_literal_as_word(String_View sv)
 {
     assert(sv.count < 1024);
-    char cstr[sv.count + 1];
+    char cstr[1024];
```

Comment:
Why not 1025 here, the extra char for '\0' ?
In fact, the previous statement was correct, it accounted for the extra char.